roles/docker/tasks/main.yml

```diff
@@ -83,7 +87,7 @@
       [Service]
       EnvironmentFile=-/etc/sysconfig/docker-local
       ExecStart=
-      ExecStart=/usr/bin/docker -d -H fd:// $other_arg
+      ExecStart=/usr/bin/docker -d -H fd:// $other_arg $DOCKER_STORAGE_OPTIONS
```

Comment:
Defined in `EnvironmentFile=-/etc/sysconfig/docker-storage`, which created by task in lvm.yml (/etc/sysconfig/docker-storage itself updated by docker-storage-setup)